index.js

```diff
@@ -184,7 +186,7 @@ AFRAME.registerComponent('teleport-controls', {
 
     if (!data.collisionEntities) {
       this.collisionEntities = [];
-      return
+      return;
     }
 
     collisionEntities = [].slice.call(el.sceneEl.querySelectorAll(data.collisionEntities));
```

Comment:
You can do a normal for loop versus creating array

Reply:
Not sure if I understand what you mean :\